js/deno.ts

```diff
@@ -41,6 +41,7 @@ export { FileInfo } from "./file_info";
 export { connect, dial, listen, Listener, Conn } from "./net";
 export { metrics } from "./metrics";
 export { resources } from "./resources";
+export { runCommand } from "./command";
```

Comment:
Consider exporting `ExitStatus` as well, so it can be referred to directly by users.